mod/config/pkg/template/template.go

```diff
@@ -20,68 +20,7 @@
 
 package template
 
-//nolint:lll // template.
-const TomlTemplate = `
-###############################################################################
-###                                BeaconKit                                ###
-###############################################################################
+import _ "embed"
 
-[beacon-kit.engine]
-# HTTP url of the execution client JSON-RPC endpoint.
-rpc-dial-url = "{{ .BeaconKit.Engine.RPCDialURL }}"
-
-# Number of retries before shutting down consensus client.
-rpc-retries = "{{.BeaconKit.Engine.RPCRetries}}"
-
-# RPC timeout for execution client requests.
-rpc-timeout = "{{ .BeaconKit.Engine.RPCTimeout }}"
-
-# Interval for the startup check.
-rpc-startup-check-interval = "{{ .BeaconKit.Engine.RPCStartupCheckInterval }}"
-
-# Interval for the JWT refresh.
-rpc-jwt-refresh-interval = "{{ .BeaconKit.Engine.RPCJWTRefreshInterval }}"
-
-# Path to the execution client JWT-secret
-jwt-secret-path = "{{.BeaconKit.Engine.JWTSecretPath}}"
-
-[beacon-kit.logger]
-# TimeFormat is a string that defines the format of the time in the logger.
-time-format = "{{.BeaconKit.Logger.TimeFormat}}"
-
-# LogLevel is the level of logging. Logger will log messages with verbosity up 
-# to LogLevel.
-log-level = "{{.BeaconKit.Logger.LogLevel}}"
-
-# Style is the style of the logger.
-style = "{{.BeaconKit.Logger.Style}}"
-
-[beacon-kit.kzg]
-# Path to the trusted setup path.
-trusted-setup-path = "{{.BeaconKit.KZG.TrustedSetupPath}}"
-
-# KZG implementation to use.
-# Options are "crate-crypto/go-kzg-4844" or "ethereum/c-kzg-4844".
-implementation = "{{.BeaconKit.KZG.Implementation}}"
-
-[beacon-kit.payload-builder]
-# Enabled determines if the local payload builder is enabled.
-enabled = {{ .BeaconKit.PayloadBuilder.Enabled }}
-
-# Post bellatrix, this address will receive the transaction fees produced by any blocks 
-# from this node.
-suggested-fee-recipient = "{{.BeaconKit.PayloadBuilder.SuggestedFeeRecipient}}"
-
-# The timeout for local build payload. This should match, or be slightly less
-# than the configured timeout on your execution client. It also must be less than
-# timeout_proposal in the CometBFT configuration.
-payload-timeout = "{{ .BeaconKit.PayloadBuilder.PayloadTimeout }}"
-
-[beacon-kit.validator]
-# Graffiti string that will be included in the graffiti field of the beacon block.
-graffiti = "{{.BeaconKit.Validator.Graffiti}}"
-
-# EnableOptimisticPayloadBuilds enables building the next block's payload optimistically in
-# process-proposal to allow for the execution client to have more time to assemble the block.
-enable-optimistic-payload-builds = "{{.BeaconKit.Validator.EnableOptimisticPayloadBuilds}}"
-`
+//go:embed config.toml.tpl
```

Comment:
I am wondering how using `embed` is beneficial as the whole template was being assigned to a variable ?

Reply:
Splitting long content template into file is more friendly for reading, also for IDE rendering.

Reply:
Thanks! @itsdevbear I would like to know your thoughts on this ?